src/plugins/recursion/README.md

```diff
@@ -0,0 +1,64 @@
+
+
+## Introduction
+
+I took [Relax NG](http://relaxng.org/spec-20011203.html#define-ref)'s `ref` and `define` part and tried to "copy" 
+their approach for elektra's specification.
+Until now I haven't really gotten a sensible idea for that though. Maybe we could talk on the next meeting about this.
+
+Also I have no idea what your `struct` plugin actually does. There is no example, just some code snippet from 
+the contract which doesn't help understanding the plugin.
+
+## Problems
+1) We want to check if all recursions are non-cyclical
```

Comment:
Details please. Which recursion do you mean? Cycles of links (override/fallback)?